src/browser/modules/EditorInfo/NodeLabel.jsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * This component is used to display the node label on editor drawer
+ */
+import React from 'react'
+import {
+  DrawerSubHeader,
+  DrawerSection,
+  DrawerSectionBody
+} from 'browser-components/drawer'
+import * as _ from 'lodash'
+
+export const NodeLabel = props => {
+  let labels = _.map(props.nodeLabel, label => {
```

Comment:
first we mapped props.nodeLabel to labels and later we mapped labels to render components.
why not directly map props.nodeLabel to render components?

Reply:
Done with changes